episodes/multi-dimensional-images.md

```diff
@@ -0,0 +1,449 @@
+
+Therefore, it's always worth checking you understand which axes are being shown 
+in Napari and what they represent! Check against your prior knowledge of the 
+experimental setup, and check the metadata in the original image (we'll look at 
+this in the next episode). If you want to change how axes are displayed, 
+remember you can use the roll or transpose dimensions buttons as discussed in 
+the [imaging software episode](imaging-software.md). Also, if loading the image 
+manually from the console, you can provide some extra information like the 
+`channel_axis` parameter discussed above. 
+
+::::::::::::::::::::::::::::::::::::: challenge 
+
+## Interpreting image dimensions
+
+Remove all image layers, then open a new image by copying and pasting the 
+following into Napari's console:
+
+```python
+from skimage import data
+image = viewer.add_image(data.kidney(), rgb=False).data
+```
+
+This is a fluorescence microscopy image of mouse kidney tissue. 
+
+1. How many dimensions does it have?
+
+2. What do each of those dimensions represent? (e.g. t, c, z, y, x) **Hint:** 
+try using the roll dimensions button ![](
+https://raw.githubusercontent.com/napari/napari/main/napari/resources/icons/roll.svg
+){alt="A screenshot of Napari's roll dimensions button" height='25px'} to view 
+different combinations of axes.
+
+3. Once you know which dimension represent channels, remove the image and load 
+it again with:
+  ```python
+  # Replace ? with the correct channel axis
+  viewer.add_image(data.kidney(), rgb=False, channel_axis=?)
+  ```
+
+4. How many channels does the image have?
+
+:::::::::::::::::::::::: solution 
+
+## Solution
+
+### 1
+The image has 4 dimensions, which we can see with:
+```python
+image.shape
+```
+
+```output
+(16, 512, 512, 3)
+```
+
+### 2
+If we press the roll dimensions button ![](
+https://raw.githubusercontent.com/napari/napari/main/napari/resources/icons/roll.svg
+){alt="A screenshot of Napari's roll dimensions button" height='25px'} once, we 
+can see an image of various cells and nuclei. Moving the slider labelled '0' 
+seems to move up and down in this image (i.e. the z axis), while moving the 
+slider labelled '3' changes between highlighting different features like nuclei 
+and cell edges (i.e. channels). Therefore, the remaining two axes (1 and 2) must 
+be y and x. This means the image's 4 dimensions are (z, y, x, c)
+
+### 3
+The channel axis is 3 (remember that the numbering always starts form 0!)
+
+### 4
+There are 3 channels which we can see from the `.shape` output, or from the 
+number of layers in the layer list.
+
+:::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+## RGB
+
+For the final part of this episode, let's look at RGB images. RGB images can be 
+considered as a special case of a 2D image with channels (yxc). In this case, 
+there are always 3 channels - with one representing red (R), one representing 
+green (G) and one representing blue (B).
```

Comment:
I think the key thing to mention somewhere in this section (especially given some of the comments about layers versus 3-D image) is that this is a special case where there is a clear way of combining the channels into one color value, instead of using a lookup table to interpret the value.